notebooks/crisp_anomaly_detection.ipynb

```diff
@@ -164,9 +164,15 @@
    "source": [
     "### Perform anomaly detection using z-scores\n",
     "\n",
-    "For each time window and product, we'll compute the z-score of `sales_quantity` and identify outliers where the absolute z-score exceeds 3.5."
+    "For each time window (in this case each month) and product, we'll compute the z-score of `sales_quantity` and identify outliers where the absolute z-score exceeds 3.5."
```

Comment:
When reading through this I wondered if we should elaborate anywhere about how time windows are chosen, but if it's just dependent on the data source, maybe we don't need to. I'm not sure if my addition of "in this case each month" is accurate here or adds value, so we can remove if needed.